extension/llm/runner/text_token_generator.h

```diff
@@ -23,12 +23,14 @@ class ET_EXPERIMENTAL TextTokenGenerator {
   TextTokenGenerator(
       Tokenizer* tokenizer,
       TextDecoderRunner* text_decoder_runner,
+      bool use_int32_token,
```

Comment:
This is causing BC breaking and causes CI failure. Can we set a default value so it's BC compatible? Also assert when it doesn't match metadata.

Reply:
The CI breaking stack trace can be found [here](https://github.com/pytorch/executorch/actions/runs/11765104349/job/32771040544?fbclid=IwZXh0bgNhZW0CMTEAAR2LBZNme_WdX6YIVmBa83nZswo_ZGZpzE101yaAzaFIJuccgjBRkZRkFPY_aem_UQNM25k9kHiR3Ceg845sog)